backend-rust/.env.mainnet

```diff
@@ -7,6 +7,7 @@ CCDSCAN_INDEXER_DATABASE_URL=postgres://postgres:secret@localhost/ccdscan_mainne
 
 # Database connection used by the ccdscan-api.
 CCDSCAN_API_DATABASE_URL=${CCDSCAN_INDEXER_DATABASE_URL}
+CCDSCAN_API_NODE_COLLECTOR_BACKEND_ORIGIN="https://dashboard.mainnet.concordium.software"
```

Comment:
Please add a comment above with some description
like: `URL for the Concordium Node Collector Backend tracking the current status of nodes` or something